src/style.css

```
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  /* height: 60px; */
+}
+
+li {
+  display: inline;
+  margin-right: 20px;
+}
```

```suggestion
li {
  display: inline;
  margin-right: 20px;
}
```
always use classes to style elements. For example
`.nav__item:not(:last-child) {`
  `margin-right: 20px;`
`}`